src/components/Board/PanZoomRotate.js

```diff
@@ -63,7 +63,7 @@ const PanZoomRotate = ({ children }) => {
 
     // Made the scale multiplicator Mac specific, as the default one was zooming way too much on each gesture.
     const scaleMult =
-      (e.deltaY < 0 ? -3 : 3 * dim.scale) / (isMacOS() ? 100 : 20);
+      (e.deltaY < 0 ? -3 : 3 * dim.scale) / (isMacOS() ? 50 : 20);
```

Comment:
Sorry @jrmi, this change is not related to current MR, but more a fine-tune after some time testing the zoom. I was lazy to create another MR just for this change...